tools/sz_repo_cli/pubspec.yaml

```
@@ -10,26 +10,26 @@ name: sz_repo_cli
 version: 0.0.1
 publish_to: none
 
+environment:
+  sdk: ">=2.10.0 <3.0.0"
+
 dependencies:
-  args: "^1.5.2"
+  args: ^2.3.1
   path: "^1.6.4"
-  http: "^0.12.1"
+  http: ^0.13.5
   async: "^2.4.1"
-  yaml: "^2.2.1"
+  yaml: ^3.1.1
   quiver: any
-  pub_semver: ^1.4.2
-  colorize: ^2.0.0
-  git: ^1.0.1
-  pubspec_parse: "^0.1.5"
+  pub_semver: ^2.1.2
+  colorize: ^3.0.0
+  git: ^2.0.0
+  pubspec_parse: ^1.2.1
   test: any
   meta: ^1.1.8
-  glob: ^1.2.0
+  glob: ^2.1.1
   pedantic: ^1.9.0
-  grinder: ^0.8.5
+  grinder: ^0.9.2
   union: ^0.0.3+1
-  optional: ^4.1.0
-  console: ^3.1.0
-  rxdart: ^0.27.1
-
-environment:
-  sdk: ">=2.10.0 <3.0.0"
+  optional: ^6.1.0+1
+  console: ^4.1.0
+  rxdart: ^0.27.1
```

```suggestion
  rxdart: ^0.27.1

```

Nit: At newline at end of file